surefire-api/src/main/java/org/apache/maven/surefire/api/util/internal/ObjectUtils.java

```diff
@@ -44,4 +46,11 @@ private ObjectUtils()
     {
         return ManagementFactory.getRuntimeMXBean().getSystemProperties();
     }
+
+    public static boolean isSecurityManagerSupported()
+    {
+        ClassLoader classLoader = ObjectUtils.class.getClassLoader();
+        Class<?> smClass = tryLoadClass( classLoader, "java.lang.SecurityManager" );
+        return smClass != null && !smClass.isAnnotationPresent( Deprecated.class );
+    }
```

Comment:
Is checking java version >= 17 not enough?

Reply:
It was my previous solution but the class appears in module surefire-booter. This is surefire-api, so we cannot inherit it and cannot duplicate the code.

Reply:
what about Java Version from commons lang3 is is accessible by `org.apache.maven.surefire.shared.lang3.JavaVersion`

Reply:
@slawekjaranowski
Now JavaVersion is in use!